Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/ParabolicTeleportPointer.cs

```diff
@@ -26,36 +27,33 @@ public class ParabolicTeleportPointer : TeleportPointer
 
         #region MonoBehaviour Implementation
 
-        protected override void OnValidate()
-        {
-            base.OnValidate();
-            EnsureSetup();
-
-            if (parabolicLineData.LineTransform == transform)
-            {
-                Debug.LogWarning("Missing Parabolic line helper.\nThe Parabolic Teleport Pointer requires an empty GameObject child for calculating the parabola arc.");
-            }
-        }
-
         protected override void OnEnable()
         {
             base.OnEnable();
             EnsureSetup();
-
-            if (parabolicLineData.LineTransform == transform)
-            {
-                var pointerHelper = new GameObject("ParabolicLinePointerHelper");
-                pointerHelper.transform.SetParent(transform);
-                pointerHelper.transform.localPosition = Vector3.zero;
-                parabolicLineData.LineTransform = pointerHelper.transform;
-            }
         }
 
         private void EnsureSetup()
         {
             if (parabolicLineData == null)
             {
-                parabolicLineData = GetComponent<ParabolaPhysicalLineDataProvider>();
+                parabolicLineData = gameObject.GetComponent<ParabolaPhysicalLineDataProvider>();
```

Comment:
Noticed this one was bigger than the others -- just making sure since we don't have test coverage here, could you check to make sure teleport pointers behave okay after this change?

Reply:
Teleportation and tooltips work - if you know of other components using parabolas let me know and I'll test them as well.